gui/gui.js

```diff
@@ -837,7 +845,7 @@ class GuiCtrl {
   }
 
   async send (...args) {
-    if (this.closed) throw Error(`Cannot send to closed ${this[kCtrl]}`)
+    if (this.closed) throw CANNOT_SEND(`Cannot send to closed ${this[kCtrl]}`)
```

Comment:
this one should be INSTANCE_CLOSED or similar

Reply:
done